lib/aes.ex

```diff
@@ -35,9 +35,11 @@ defmodule Fields.AES do
     key_id = get_key_id()
     key = get_key(key_id)
     {ciphertext, tag} = :crypto.block_encrypt(:aes_gcm, key, iv, {@aad, to_string(plaintext), 16})
-    key_id_str = String.pad_leading(to_string(key_id), 4, "0") # 1 >> "0001"
+    # 1 >> "0001"
```

Comment:
I really wish `mix format` would leave in-line comments alone ... 
moving them to the next line loses the context. 💭 

not the fault of this PR, just the formatter being unhelpful. 👍